### ISSUE TITLE
Outline for future LuV-UV quests

### ISSUE THREAD

**Issue by chochem** [GTNewHorizons/GT-New-Horizons-Modpack#6161]
-- updated 07 June --

Ok, so @Prometheus0000 wanted to know a bit more details of the progression line for future quests starting from the current LuV questbook onward.

If only 2-3 quests are added it would be best to just turn the LuV tab into a LuV+ tab, but I am thinking a bit bigger here, so we should probably have dedicated ZPM and UV tabs. At most this should go up to UHV. But an extra Endgame Goals tab beyond that is a really good idea: https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/6172

My general idea would be that these quests do not ask for single block machines where multis are available, and generally focus on new machines and less obvious concepts.

related issues: https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/6128, https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/6130, https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/6115, https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/6127

Edit: I also like the idea of integrating the *Fusion*-tab as well as the *An Age of Tech*-tab, as those only have a very small number of quests.

---
**LuV Tab**

- a quest early in the LuV tab to make purple rocket fuel (CN3H7O3) needed for T5/T6 rocket (probably better space race tab)

- a quest to make an LuV energy hatch as those are much more important than any machines in this tier. Should be behind the existing LuV-SC wire quest and behind LuV assembler. (add as requirement for starting fusion)

- a quest early in the LuV tab to obtain a small amount of trinium from deep iron (small ore!) and make a bit of naquadah alloy. (add as requirement for starting fusion)

Integrate quests from Fusion tab:
 - starting fusion. (move over) Correct typo: each energy hatch only has a capability of 10MEU (not 160MEU)

 - *Making power with your plasma in LuV*. move over and add the 'in LuV' to the name. Might also need an overhaul, the best option for a dynamo is a buffered LuV Dynamo (the quest talks about UV and demands a normal LuV dynamo) 

- *Turbine time in LuV*. move over and add the 'in LuV' to the name. Change the text to talk specifically about materials available at this stage. Best option imo: normal draconium turbine.

- Getting europium. move over, and update the text: t6 rocket should be t5, and only 'as part of a rare ore' maybe. Also point out that this material is key to T5 rocket and to ZPM. (add as requirement to unlock ZPM)

continued:
- Fusion Rocketry (see comments)
- additional quest to collect a large amount of Enceladus Ice, and turn that into mysterious crystal dust. The dimension also has crystals to be harvested for unknown crystal shards, but the ice is much better. 

 - change IV/LuV/ZPM/UV crystal circuit quests to only make one circuit and with the now more expensive recipe (this circuit is for imprinting the CAL)

CAL (=circuit assembling line) quest line:
- Collect Prasiolite, (cubic) Zirconia, Tellurite. The quest should point to the EBF recipes for flawed crystals, as those are hard to find. Flawed Prasiolite from Amethyst Dust and flawed cubic zirconia from zirconium+yttrium oxide. 
Tellurite should normally be from Temagamite. But if that is not available in the current patch, there is also alburnite from rare earth.

- Then use that to make Magneto Resonatic Dust, Circuit Compound Mk3 and Exquisite Prasiolites
- Slice the Crystal Circuit and make imprints. usage: put into CAL controller. this is permanent. this CAL will only produce this one circuit.
- craft up 4 CALs  
- change the *cheaper HV circuit*-quest. this should be about making a 5th CAL, imprinting it and using the new recipe.
- add Quantumprocessor (EV) ASoC quest for 6th CAL (use 2 LuV energy hatches)

---

**ZPM Tab**

- ZPM machine hull.
- *you cant get enough of this 1-4 in ZPM*.
- *you cant get enough of this 5-6 in ZPM*.
- *Superconductors 131072 eu/t*.
- ZPM assembler
- Tier VII field gen. (add as requirement for Mk2 fusion)
- Europium doped Boule/Wafer (needed for NPIC Wafer for Mk2 fusion)
- ZPM energy hatch.  Should be behind the existing SC wire quest and behind ZPM assembler. (add as requirement for Mk2 fusion)
- Crystal SoC for cheaper Crystalprocessors (IV) (use 2 zpm energy hatches)
- everglades portal&dimension, see https://github.com/alkcorp/GTplusplus/wiki/The-Toxic-Everglades and https://github.com/alkcorp/GTplusplus/issues/127 for some infos, 10 frame blocks for the portal suffice. (behind ZPM energy hatch due to Pikyonium 64B)
- 80 vigorous Laurenium Machine Casing. This is a longer quest or multiple. Besides Ytterbium for Pikyonium which was already needed before, it also needs Rhenium (ABS recipe from scheelite, molybdenum, or molybdenite) and Dysprosium (rare earth III or everglades for Lepersonnite). It is needed for green rocket fuel (behind ZPM energy hatch due to Laurenium and Pikyonium 64B) (maybe better in the gt++ tab?)
- green rocket fuel (H8N4C2O4), needed for T7/T8 rocket (probably better space race tab). This quest line exists already in HV! However it can not be completed there as Laurenium casings are required making it ZPM+. Actually it might be best to move that entire quest line to the space race tab and extend it with the other rocket fuels.

more fusion quests:
- *Fusion reactor MK2 - bigger, better, fusor* (just move quest over)
- *Making americium*. Move quest over and point out that americium is a key material in UV. (add as requirement for UV tab)
 - *Making power with your plasma in ZPM*. asks for a buffered ZPM dynamo hatch and points out the advantages of tin fusion.
- *Turbine time in ZPM*. Quest about turbine options in ZPM. Good options imo: small trinium, large draconium, large neutronium, large oriharukon, large ichorium
- Tritanium. Lets make some Tritanium in the mk2 fusion reactor. useful for: T6 rocket, Gravisuit (has quest), possible tool material, drilling plant and void miner 1

continued:
- Void Miner I. note that you can get neon for the tier I easily from bartimaeusnek dust. Krypton for tier II and Xenon (not for a recipe but an option for running it) however need a nuclear reactor or the DT liquid air recipe. Oganesson for tier III needs mk3 fusion. All can be replicated with uum.

- Make some raw (or sterilized) growth medium. assuming new biolab and vat quests have been added somewhere previously. this is needed for wetware circuits
- Create stemcells. Either from chickens (LuV quest) or from growth medium and unknown crystals.

wetware questline:
- ZPM circuit assembling machine (needed for T7 rocket and wetware circuits)
- wetware lifesupport boards
- extreme wetware lifesupport circuit boards
- wetwareprocessor (LuV) (again, just ask for one circuit the expensive way, expect the player to imprint a CAL with it)
- wetwareprocessor assembly (ZPM)
- wetware supercomputer (UV)  Best mention that the mainframe only becomes available in UV.

and continued:

- T6 rocket, should be in space race tab (note that this needs the gt++ large scale auto-assembler to be crafted in ZPM. That multi has a quest already and the recipe is ZPM-tier. Otherwise, this can just be postponed to the UV assembler)
- Where No One Has Gone Before (see comments)
- Smelting neutronium: neutronium is needed in UV and for the T7 rocket, so lets make some! (add as requirement for UV tab)

---

**UV Tab**

- UV  machine hull.
- *admit it, you love this 1-4 in UV*.
- Ultimate Time Anomaly (highest tier WA)
- *admit it, you love this 5-6 in UV*.
- *Superconductors 524288 eu/t*.
- Tier VIII field gen. (add as requirement for Mk3 fusion)
- (no UV assembler quest or just an optional one, as the multiblock is available since ZPM)
- UV energy hatch. (add as requirement for Mk3 fusion and tectech quests)
- Americium doped Boule/Wafer (needed for QPIC Wafer for Mk3 fusion)
- T7 rocket, should be in space race tab  (can also be done later, o new materials or anything like that)
- All Good Things... (see comments)

TecTech Quests:
- *Learning curve? Where we're going there are no curves.* move over.
- The Beginning of the End Game. move over, maybe also change the casing counts. The numbers are completely arbitrary. I had 58 high power casings, 56 adv. com. cas. and 7 comp. cas. left over after building the quantum computer, research station, and data bank, but had too few fiber cables. (not counting quest rewards) I should also point out that the explanation of the uncertainty resolver is completely useless and misleading, however, there is a better explanation on the tectech-wiki.
- Quantum Computer. move quest over. Add important info, that you can copy paste the middle segment multiple times to make the multiblock longer and allow for more computer racks. This will be crucial later on to get more computation/s. The quest should also ask for (at least) 2 computer racks.
- Research station. Move quest over, add info to place the controller at least 3 blocks above the ground.
- New Data Bank quest: ask for 1 data bank, 1 assembly line slave connector, 1 data bank master connector, 18 computer heat vents, 6 high power casings. The number of computer casings and (advanced) data hatches is variable.
 Also mention that it takes a maintenance hatch and at least an IV energy hatch. (it might be 2048 + (amount of master optics * amount of data hatches * 2048). scan to be sure)

continued:
- wetware mainframes (UHV)
- Solar Panel LuV (needs wetware mainframes)
- singular solar panel
- uv circuit assembling machine (needed for T8 rocket and bio circuits)

more fusion quests:
- *upgrading the reactor* (move over from fusion tab). Change text, it can not do all fusion recipes. Instead write: The reactor can handle recipes up to 640M EU start up.
 - *Making power with your plasma in UV*. asks for a buffered UV dynamo hatch. tin is still top, but also mention americium plasma, maybe others?
- *Turbine time in ZPM*. Quest about turbine options in UV. Good options imo: large/huge trinium, large/huge cosmic neutronium
- *Time for another reactor* (move over from fusion tab)
- *Making more americium* (move over from fusion tab)

- maybe T8 rocket here? currently uncraftable

continued:
- Preparations for UHV: Make some Bedrockium (from Cobblestone!, ore with T8 rocket) and Cosmic Neutronium (from Black Plutonium)

---

**UHV Tab**
- UHV  machine hull.
- *... this 1-4 in UHV*.
- maybe T8 rocket here? currently uncraftable
- *Superconductors 2097152 eu/t*.
- UHV energy hatch.
- maybe T8 rocket here? currently uncraftable
- The Final Frontier (where T8 rocket can be crafted) (see comments)

- *... this 5-6 in UHV*.
- Tier IX field gen.

Avaritia/Infinity quest line:
- neutronium compressor
- infinity catalyst
- infinity ingot
- neutron collector

Bioware Circuits quest lines:
- TCetiE-Seaweed Extract
- Raw/Sterilized Bio medium
- Bio Cells
- Ultra Bio Mutated Circuit Board
- Bio Processing Unit
- Bioprocessor (ZPM)
- Biowareprocessor Assembly (UV)
- Bioware Super Computer (UHV)
- Bio Mainframe (UEV), needs 256 Computation! (importan: you can chain Quantum computers, or extend the middle section to allow for more computer racks)
- ZPM solar
- Light-Absorbing Solar Panel

(these two questline should be independent)

---

**Endgame Goals Tab**

refer to: https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/6172

---

**Others Things I have not yet looked into**

- single block plasma gens
- details/order for UHV tab
- electric implosion compressor

Let me know if you see a mistake or something that should be added because its mandatory or just useful and unlocks in these tiers.

**Comment by combusterf** [comment]
Some additions:
- I'd drop the fusion tab altogether, that can go into the 3 tiers.
- I would keep sensors/emitters and later field generators in dedicated quests - a very good first machine is the assembler which doesn't need the investment of sensors/emitters, same with field generator, which you only really need for the fusion computer (and the CAL in LuV) but not for the single block circuit assembler or laser engraver
- There should be info on how much power certain options actually generate so people can plan an approach. There should probably some extra information on the single block plasma and naquadah generators that are currently a bit obscure. Per-tier dynamo hatch quests with associated plasma flow and rotor information would also be pretty helpful as documentation.

**Comment by Regian24** [comment]
> Superconductors 131072 eu/t.
wait, isn't UV 524288?

**Comment by Regian24** [comment]
also I think the rocket fuels quest should go in the space race tab

**Comment by chochem** [comment]
> 
> 
> also I think the rocket fuels quest should go in the space race tab

sure, I am happy with that.

**Comment by Regian24** [comment]
> sure, I am happy with that.

and the rockets quest go in space race too.
the only "space" quests that are going to be in the LuV-ZPM-UV tab should be the "Dimensional Goal" quests, and the quests for the new ores.

**Comment by chochem** [comment]
I still keep them in here. so that prometheus has an idea of the order

**Comment by Regian24** [comment]
> I still keep them in here. so that prometheus has an idea of the order

okay okay

**Comment by Prometheus0000** [comment]
Also, Everglade portal+disk+dim. Is there anything _only_ gotten there that's useful? I think it's ZPM (pikyonium).

**Comment by chochem** [comment]
added. yes its ZPM. I might add more details soon, Just started crafting the portal.

**Comment by draknyte1** [comment]
Any materials that were Everglade specific can now be obtained some other way. Like ytterbium.

**Comment by Prometheus0000** [comment]
Could you suggest some LuV turbine materials? Keeping in mind they were changed a few weeks ago.
Also, making power suggests that helium is renewable from water somehow (not the deuterium part)?
Was it just the bartimaeusnek ore that had europium in it?

**Comment by Prometheus0000** [comment]
Also, can you get the t5 rocket before the CAL to get the Prasiolite ore from ross128ba?
And does the Temagamite ore for Tellurium spawn anywhere now? It was on the moon, and bart said it should be on mercury, but NEI says nothing now.

**Comment by chochem** [comment]
1. based on buffered dynamo hatches:
LuV turbines: normal draconium, I dont see any reasonable competition
ZPM turbines: small trinium, large draconium, large neutronium, large oriharukon
UV turbines: large/huge trinium, large/huge cosmic neutronium
later: awakened draconium, infinity catalyst, infinity

disclaimer: I have not personally tested all these options and might be missing some too. I do not take responsibility for any turbine explosions caused by these recommendations. :P

2. I havent really done'much he-plasma tbh. But He-3 can be pumped, it is also later needed for tin fusion. And Deuterium is either from centrifuging air (but not much) or from Hydrogen (much better). Of course you can get Hydrogen from water but that is only one option. The comment isn't to clear to me either.

3. yeah, that supposed T6 europium ore is a mystery to me. Must have been changed quite some time ago or I am missing something big. Afaik there is only fusion, uum, the bart vein (tertiary+centrifuging dust, T5), vega b stone dust (T8), and DD void miner as options to get europium.

**Comment by Prometheus0000** [comment]
I believe there's also Ichorium turbines even if it's not on the spreadsheet.

**Comment by chochem** [comment]
ah yes. ichorium is actually really good in zpm. Of course thats only reasonable if you have a setup to make ichorium in a reasonable decent fashion.

**Comment by chochem** [comment]
> 
> 
> Also, can you get the t5 rocket before the CAL to get the Prasiolite ore from ross128ba?

yes. you just need europium for the rocket and the fusion reactor *just* needs the basic LuV components and ZPM circuits (UV circuits are only needed for T2 fusion, UHV circuits for T3 fusion)

> And does the Temagamite ore for Tellurium spawn anywhere now? It was on the moon, and bart said it should be on mercury, but NEI says nothing now.

yeah it used to be bugged. I dont know any more than what NEI says about the current status.

the important thing to point out in a quest are definitely the slightly hidden EBF recipes: Flawed Prasiolite from Amethyst Dust and flawed cubic zirconia from zirconium+yttrium oxide.

Tellurium also has a more tricky secondary option in case Temagamite is unavailable: rare earth III for alburnite. (it can be replicated, so only one Tellurium is needed)

**Comment by Prometheus0000** [comment]
I hate text (sometimes)
![image](https://user-images.githubusercontent.com/8517039/83623547-787aa080-a55f-11ea-906d-5b28c00dc531.png)

I'll go check out mercury.

**Comment by Regian24** [comment]
Wow that quest looks really cool

**Comment by Regian24** [comment]
But, instead of a wall of text, what if we separate the 3 materials into three quest?

**Comment by Prometheus0000** [comment]
Yeah, there isn't any there or on the moon, or on ross128b or ross128ba. @bartimaeusnek did you decide to remove it from worldgen?

It's still a WIP, do other people think it should be separated? Keeping in mind I still have to explain Tellurite.

**Comment by Dream-Master** [comment]
the small ores and ores not working properly. I found this ores on all planets on moons so I decided to disabled it.

**Comment by Prometheus0000** [comment]
Did you just disable generation in new chunks or will it delete existing ores?

**Comment by bartimaeusnek** [comment]
Well it generated everywhere when it should've been only on mercury... So the generation is disabled for now, we maybe need a dimwipe on the officials... 

Will look into it asap.

**Comment by Prometheus0000** [comment]
It's kind of weird that the plasma multi uses UV machine hulls, is it normally UV? That would make the single blocks less in a weird place.

**Comment by Prometheus0000** [comment]
Ok, serious question here, how do you get Deep Iron for trinium? You can get it from T5 planets, but that needs the T5 rocket, and that need Europium, which needs Fusion, which needs Naq. Alloy, which needs trinium, which needs deep iron.

The planet dusts that have it are also T5. Though if you have space flowers you can make it. I'm 100% sure that's not the intended way though. I guess this is an oversight from making the T5 rocket require Europium bolts? Or did I miss something?

**Comment by Prometheus0000** [comment]
Ok NM I didn't look at the small ore page.

**Comment by Regian24** [comment]
It's a small ore I think? I hven't reached that part of the game myself

**Comment by combusterf** [comment]
> Ok, serious question here, how do you get Deep Iron for trinium? 
It exists as a small ore on Mercury if it hasn't been changed since. The ore drill works fine for this, and two runs were sufficient for fusion and a good part of ZPM's needs.

**Comment by Prometheus0000** [comment]
Things to add: Void miners, more planet grab the ore quests (and just going there), solar panel/block quests.

**Comment by chochem** [comment]
> 
> 
> Things to add: Void miners, more planet grab the ore quests (and just going there), solar panel/block quests.

added void miner, the recipes are a bit weird at the moment, so mk4 miner, void miner I, and void miner II unlock together (maybe upto obtaining krypton)

add solar panel LuV. Though that recipe is completely f*d. the only reason to make one of these is for memes or pack completion, you would need like 400 of them (i.e. 2400 wetware mainframes, etc) to keep up with the mk3 fusion that you unlock at the same stage (which only needs 12 wetware mainframes)

I would be against grab the ore quests, as people just go with LuV prospector scanner+ multiblock miner (crushing included) at this stage. So *reach dimension* quests would be better imo.

**Comment by Prometheus0000** [comment]
How awkward. I would have figured that the VM I would be way slower to account for it being infinite, so it would be a sidegrade, not an upgrade. but being able to get the 2nd right away is weird. I want people to be aware they exist though.

We also need the ZPM and UV one, we should list them all, UV can just be on two tabs. Personally I don't like passive power production since it seems unbalanced (solar, kinetic), but I think producing T-2 power out would be better than making the recipes incredibly grindy. Though on second look I guess you're complaining that they don't put out enough power. Or that fusion is too good.

Shouldn't we tell them what new ores are available? I'd be fine with a combined X-rocket tier planets+64 of each new ore. For example for T7 it would be: location tasks for Haumea, Kuiper Belt, Makemake, and Pluto. And collect 64 of Black Plutonium Ore, Red Garnet Ore, Yellow Garnet Ore, Borax Ore, Chrysotile Ore, Realgar Ore, Garnet Sand, Nether Star Ore. All in the same quest

And I wonder if planet dusts are actually *required* at some point?

**Comment by Prometheus0000** [comment]
I see what you mean now, the ores you pull up would be crushed already.

**Comment by GTNH-Colen** [comment]
> We also need the ZPM and UV one, we should list them all, UV can just be on two tabs. Personally I don't like passive power production since it seems unbalanced (solar, kinetic), but I think producing T-2 power out would be better than making the recipes incredibly grindy. Though on second look I guess you're complaining that they don't put out enough power. Or that fusion is too good.
> 
Fusion will probs be nerfed down the line, there are already tickets about this but solar is 100% too expensive atm after IV it becomes totally useless due to the massive cost increase.

> Shouldn't we tell them what new ores are available? I'd be fine with a combined X-rocket tier planets+64 of each new ore. For example for T7 it would be: location tasks for Haumea, Kuiper Belt, Makemake, and Pluto. And collect 64 of Black Plutonium Ore, Red Garnet Ore, Yellow Garnet Ore, Borax Ore, Chrysotile Ore, Realgar Ore, Garnet Sand, Nether Star Ore. All in the same quest
> 
I really despise quests forcing you to go gather an ore, in the late game you should know how to use NEI and if you don't then tough honestly. The quests forcing you to get the *actual* ore later on are even more atrocious because as said before everyone is using a miner at this point. If you do add "go to dim" quests then please make sure they are absolutely required, it is infuriating flying to a dimension to immediately leave just for a quest.

> And I wonder if planet dusts are actually _required_ at some point?
I've never needed them for anything, they can be useful sometimes but are by no means required. Energy wise they also take a ton compared to just slapping down a miner and getting far more resources far quicker.

**Comment by Prometheus0000** [comment]
I've been working on the text outside the game, here's what I've got on the planet/rocket/ore/planet block quests, it should be done unless there's materials I'm missing like the water.
#
Fusion Rocketry
So you completed your Fusion Reactor and made your rocket, great! Now time to get some new ore! (like that isn't getting old) You don't need your hand held anymore, so I'll just list what's new.

Enceladus, Miranda, Oberon, Ross128ba and Titan are the new planets. Miranda is the 1st moon of Uranus, while Oberon is the last. Enceladus is the 2nd moon of Saturn, while Titan is the second last. Ross128ba is Ross128b's moon in the Ross128 System to the South.

The new veins are Tungsten, Nickel, and Vanadium. The ores to get are: Chrome, Deep Iron, Iridium, Gold, Magnetite, Nickel, Osmium, Silicon, Shadowiron, Tungsten, and Vanadium.

You can get Neutronium from small ores on Enceladus, Oberon, and Titan.

And grab some planet blocks while you're at it too.

You can get the schematic for T6 on Enceladus.

#

Where No One Has Gone Before
Another rocket, another set of planets. Get to work already, that's why you made it, right?

There's only two new planets this time, Proteus and Triton. They're both moons of Neptune.

The new veins are Neutronium, Niobium, and Thorium. The ores are: Adamantium, Gallium, Naquadah, Neutronium, Niobium, Plutonium-241, Titanium, Uranium 235, Uranium-238, and Yttrium.

You can get Black Plutonium from small ores on Triton.

You should pick up some planet blocks while you're here.

The schematic is on Proteus.

#

All Good Things...
So you've built your T7 rocket, great, there's only one more left. Until then, the new places you can visit are: Haumea, Kuiper Belt, Makemake, and Pluto.
Pluto is just before the Kuiper Belt, and Haumea and Makemake are just after.

The new Veins are Black Plutonium, Red Garnet and Garnet. Pick up some ore from each: Black Plutonium, Borax, Chrysotile, Garnet Sand, Nether Star, Realgar, Red Garnet, and Yellow Garnet.
 
Don't forget the planet blocks!
 
The last schematic is on Pluto. This is also the last dungeon.
 
 #
 
The Final Frontier
There's no more rockets to make after this, though there's still at least one more place to visit. After this is the Deep Dark, which has all the ores+more.
Although you get access to six new planets, there's only two new veins. And don't complain that Barnarda should be Barnard, we can't fix it.
The Barnarda system is to the NNW, and you can visit Barnada C, E, and F.
The Alpha Centauri system is to the NE, and you can vist Alpha Centauri Bb (Proxima Centauri is not visitable).
The Tau Ceti system is to the SSW, and you can visit the habitable planet Tau Ceti E.
The Vega System is to the WNW, and you can visit the planet Vega B.

The veins are Samarium and Cadmium. The ores are Cadmium, Caesium, Cerium, Lanthanum, Neodymium, Samarium, Tartarite, and Tartarite.

You can get Awakened Draconium from small ores on Barnarda E, Barnarda F, Tau Ceti E, and Vega B.
You can get Bedrockium from small ores on Barnarda F.
You can get Infinity Catalyst from small ores on Vega B.

In addition to planet ores and blocks, grab some Barnarda C Saplings and Dirt so you can grow some more if you need it. You can also grab some more Unknown Crystal Shards from the Rainbow Crystals lying around. More importantly you should collect Seaweed from Tau Ceti E. You need it to make UEV circuits, and though you can make it as well, you might want to skip that part until you automate it. The water can be found as a pumpable fluid on Tau Ceti E.

**Comment by Prometheus0000** [comment]
It should be visit dim and grab some planet blocks (16? 64?) for the most part. It won't require you to get the water, crystals, logs, or dirt. And is the logs only use to make liquid DNA, for which there are easier alternatives? Or can you use it in the bio lab to make something?

**Comment by chochem** [comment]
> 
> How awkward. I would have figured that the VM I would be way slower to account for it being infinite, so it would be a sidegrade, not an upgrade. but being able to get the 2nd right away is weird. I want people to be aware they exist though.

the t4 miner is part of the recipe of the T1 VM, which is part of the recipe of the T2 VM, but the biggest gate is the tritanium in the t4 miner. apart from that I agree that the VMs would be a bit in parallel. Anyway, I did add it at the point you obtain tritanium as indeed the mechanic should be pointed out with a quest and maybe something should be said about the different gasses. 

> 
> We also need the ZPM and UV one, we should list them all, UV can just be on two tabs. Personally I don't like passive power production since it seems unbalanced (solar, kinetic), but I think producing T-2 power out would be better than making the recipes incredibly grindy. Though on second look I guess you're complaining that they don't put out enough power. Or that fusion is too good.

I am mostly complaining that they are waay to expensive and also gated way to late. unlocking the 30k eu/t solar together with T3 fusion (9-12MEU/t) with a single solar panel barely being any cheaper is just a joke. But that should be a discussion for somewhere else really. the ZPM one is in tbc at end that I havent filled in yet (Bio Mainframes!) and the UV one should be on the endgame quest tab.


> Shouldn't we tell them what new ores are available?

absolutely. I like your drafts! except for the *Pick up some ore from each* for the reasons already discussed.

> 
> And I wonder if planet dusts are actually _required_ at some point?
not for T1-T7. but they can be very useful, see enceladus.

**Comment by Prometheus0000** [comment]
I mostly want the solar quests because they're on all the previous tiers.

That text was left over from before I realized you would be getting crushed ores. I changed it.

#4890 I'm thinking of this when I wonder if they're necessary at some points, given that Piky said so. But it's certainly possible that's no longer the case.

I realized the use of the logs. **Raw Radox**. I think 4 saplings, 4 dirt,  and 8 logs is enough. Maybe it'll be non-meme at some point, but that time is not now.

**Comment by Prometheus0000** [comment]
> T6 rocket, should be in space race tab (note that this needs the gt++ large scale auto-assembler to be crafted in ZPM. That multi has a quest already and the recipe is ZPM-tier.

Do you have to make the T6 rocket to reach UV? And thus need to make the multi to progress?

**Comment by chochem** [comment]
no you dont need it. If you dont want to make the multi you just wait til you have a UV assembler.

**Comment by Regian24** [comment]
> The ZPM tier. Speaking of ZPM, did you know the name is based on the Zero Point Module from the show Stargate SG-1, which continues the story of the movie? It's a great show, go watch it! You can also get one for completing the Legendary Quests tab.

Speaking of stargate...

**Comment by Regian24** [comment]
Ok here's my ideas for some of these:

Tier 5 tab description changed (Because the current one is crap)
Welcome to EV! You have successfully went to the moon and processed Titanium. You are now getting extreme levels of energy. In this tier you will unlock Nuclear fission energy, and Applied Energistics 2.

Tier 6 tab description (Because the current one is crap.)
You have successfully been to Mars and acquired Tungstensteel. To progress to the next tier, you'll need to make an assembly line. Good luck! 

Tier 8 tab description
Congratulations, you have completed the assembly line and fusion reactor and made it to the Zero Point Module tier. You're well into the late game now, but there's still a lot of stuff to do. Good luck!

Tier 9 tab description
Ultimate Voltage! This is the final tier in normal GregTech, but here in GregTech: New Horizons, we modified GregTech to have 6 more tiers after this. (UHV, UEV, UIV, UMV, UXV, OpV, MAX.) You have done something that only a few players are capable of doing. Welcome to the Endgame!

Tier 10 tab description
Uhh... hi? You really aren't supposed to be here, you know? We're still, you know, making this tier, right?

**Comment by Regian24** [comment]
I'll go to sleep and make some more tomorrow

**Comment by chochem** [comment]
did some more updates:
- added late UV + UHV stuff
- added more details to green rocket fuel (the one for T7/T8 rockets) and Laurenium. But the rocket fuel quests should all go to the Space Race Tab imo. The Laurenium stuff probably to the gt++ tab.
- added Ultimate Time Anomaly as optional quest (highest tier WA)

**Comment by Regian24** [comment]
> So above this is the UIV, UMV, UXV, and OpV wires. Only UIV has a cable version, and there are no superconductors. You can use UHV superconductors, or use a super cheap material and accept some losses to move your power around.
The materials to make the higher tiers are currently Nether Star, Quantium, Black Plutonium, and Awakened Draconium.

UEV tier cable is draconium btw

**Comment by Prometheus0000** [comment]
> UEV tier cable is draconium btw

I guess I forgot that one.

> Turbine time in ZPM. Quest about turbine options in ZPM. Good options imo: small trinium, large draconium, large neutronium, large oriharukon, large ichorium

I thought the large ones were always the best?

> note that you can get neon for the tier I easily from bartimaeusnek dust

I thought that ore was super rare? Because it has europium in it? You also get some from the liquid air distillation method.

> Smelting neutronium: neutronium is needed in UV and for the T7 rocket, so lets make some! (add as requirement for UV tab)

You don't use the Neutronium Compressor? Wait, it's the Neutron Collector. And there's SpNt, Nt, and the avaritia one.
You seem to get Nt from the ore (as well), and make the avaritia one from SpNt? I guess the avaritia one and the SpNt one are counted as the same? You can get piles of neutrons (to collect into ingots) from centrifuging stuff, or the collectors. From discord it seems the collectors are super slow, so maybe just add an optional quest (in UV?).

I guess in UV you can get the ore instead of making in through fusion, and can get SpNt two ways?

> Preparations for UHV: Make some Bedrockium (from Cobblestone!, ore with T8 rocket) and Cosmic Neutronium (from Black Plutonium)

You probably should be required to make them into cables. What's the SpNt used for? And what's the suggested method of getting it? The stuff I mentioned above?

> The Deepest Depths
After crafting the Tier 8 Rocket, you have discovered that you can create a portal to the core of the earth. Legend has it that this place, has all the ores in the game. Make a Portal to the Deep Dark

There's already a quest for the portal, it's on the second TC page

#
People don't like my memetastic non-descriptions? :( 
I figured making them be funny instead of boring would be fine since you should know generally what to do if told what to make.

**Comment by Regian24** [comment]
I like when the quests are funny instead of boring

**Comment by Prometheus0000** [comment]
Oh, I thought you were making boring replacements for my text, I didn't pay enough attention. And I'm still sleepy (excuse).

> Circuits, Assemble!
To make a Tier 7 Rocket, and Wetware Circuits (UHV-Tier Circuits), you need to make a ZPM Circuit Assembling Machine.

> Pick-n-Place all over the place
To make the Tier 8 Rocket, and Bioware Circuits, you need to make an UV Circuit Assembling Machine. This is the last Circuit Assembling Machine in the game

Like these ones. Though that's fine, I guess. They are more descriptive of what you can make with them. I still want a line about needing to make them in though. I bet some people would think you wouldn't need to make them anymore.

> Tier 8 tab description
Congratulations, you have completed the assembly line and fusion reactor and made it to the Zero Point Module tier. You're well into the late game now, but there's still a lot of stuff to do. Good luck!

You make the assline to get to luv (motors, etc. require it), and t8 is zpm

**Comment by Regian24** [comment]
That's what i said, i was talking about zpm

**Comment by Regian24** [comment]
The Board that Keeps Life
To make Wetware Circuits, you'll need to make a Wetware Lifesupport Circuit Board. Making this board is quite a different process than making all the other Circuit Boards you made in your playthrough, because making it actually requires materials!

You need 16 multilayer boards, 1 petri dish, 1 LuV Pump, 1 LuV Sensor, 1 IV Circuit, 16 Niobium Titanium Foil and 4000L of Sterilized Growth Medium in a LuV Circuit Assembler!

Caution: Handle with care
Make an Extreme Wetware Lifesupporting Board. I guess, by now, you know how to etch a circuit board, so I won't tell you and you have to use NEI :P

Intelligent Circuits
Another component for the Wetware Circuits is NeuroCPUs. To make it you need to use the Assembly Line. 
I won't tell you the materials, but I'll tell you that you need 250L Sterilized Growth Medium, 250L UU Matter, and 1000L of IC2 Coolant at 80,000 EU/t.

**Comment by Prometheus0000** [comment]
Do you really need to mention the assline though? But it doesn't really matter. So you think the T7 text is good?

And like I said, we don't need to list the materials off for people, they can see it in the quest book on the right. Also, I want the board and completed board as the same quest, like in LuV.

**Comment by Regian24** [comment]
> Do you really need to mention the assline though? But it doesn't really matter. So you think the T7 text is good?

What T7 Text?

**Comment by chochem** [comment]
> > Turbine time in ZPM. Quest about turbine options in ZPM. Good options imo: small trinium, large draconium, large neutronium, large oriharukon, large ichorium
> 
> I thought the large ones were always the best?
> 

Its more complicated. small/normal is better if your hatch cant support the large one. (draconium in LuV, trinium in ZPM) And huge is better (in UV) if you care more about throughput than efficiency. basically high throughput = fewer turbine multis, high efficiency = fewer fusion reactors.

> > note that you can get neon for the tier I easily from bartimaeusnek dust
> 
> I thought that ore was super rare? Because it has europium in it? You also get some from the liquid air distillation method.

it is fairly rare, but you have the LuV scanner so its not hard. Its also fairly small, but you only need 20 buckets or so for the recipe.

> 
> > Smelting neutronium: neutronium is needed in UV and for the T7 rocket, so lets make some! (add as requirement for UV tab)
> 
> You don't use the Neutronium Compressor? Wait, it's the Neutron Collector. And there's SpNt, Nt, and the avaritia one.

> You seem to get Nt from the ore (as well), and make the avaritia one from SpNt? I guess the avaritia one and the SpNt one are counted as the same? You can get piles of neutrons (to collect into ingots) from centrifuging stuff, or the collectors. From discord it seems the collectors are super slow, so maybe just add an optional quest (in UV?).
> 
> I guess in UV you can get the ore instead of making in through fusion, and can get SpNt two ways?
> 
> > Preparations for UHV: Make some Bedrockium (from Cobblestone!, ore with T8 rocket) and Cosmic Neutronium (from Black Plutonium)
> 
> You probably should be required to make them into cables. What's the SpNt used for? And what's the suggested method of getting it? The stuff I mentioned above?


neutronium = gt neutronium from ore/small ore. getting the dust is trivial, you get thousands from small ore alone. the smelting is the hard part: combine with helium plasma and then blast for 3.2 Billion Eu (with radon).

SpNt = cosmic neutronium = avaritia neutronium
you get it first from centrifuging black plutonium (and then combining 81 piles of neutrons in the implosion compressor). Later on you can get it from neutron collector+UV WA combo, but that needs infinity.

The neutron collector needs infinity ingots, see UHV tab.

SpNt is used for: UHV components, great turbines, making infinity ingots, many late game crafts

**Comment by Prometheus0000** [comment]
![2020-06-06_04 55 46](https://user-images.githubusercontent.com/8517039/83940441-08178d80-a7b2-11ea-941f-43a3788ff0ee.png)

**Comment by Regian24** [comment]
Ohh okay

**Comment by Regian24** [comment]
I think it's good. But I don't want to put my text in the "Tier 8" quest, I want it to be in the description of the tab

**Comment by Prometheus0000** [comment]
Oooooh, I forgot those even existed.

**Comment by Regian24** [comment]
I finished the wetware line, check it if it's good

**Comment by Prometheus0000** [comment]
> Wetware Mainframe
The Wetware Mainframe is the final Wetware Circuit, and it is the first UHV Circuit you can get. You need an Assembly Line at 300,000 EU/t.

> wetware supercomputer (UV) Best mention that the mainframe only becomes available in UV.

@chochem can't you send 4A over 2 hatches? What makes it UV?

**Comment by chochem** [comment]
yeah sorry should have clarified: its the tectech stuff that pushes it to UV, not the assembly line. (4A over 2 hatches is indeed fine)

**Comment by Prometheus0000** [comment]
Ok, it needs a note that you need to finish the research station in the UV tab first then.

**Comment by Regian24** [comment]
> Ok, it needs a note that you need to finish the research station in the UV tab first then.

okay adding it

**Comment by Regian24** [comment]
since all the other quests are colored (the LV ones are purple, MV is yellow, etc.), what color will we use for ZPM/UV/UHV?

**Comment by Prometheus0000** [comment]
Dunno, I figured I'd let dream figure it out. I don't even know how to set the color of quest titles.

**Comment by Regian24** [comment]
just use the color tags (like §7) and to make them bold, do §l

**Comment by Prometheus0000** [comment]
![image](https://user-images.githubusercontent.com/8517039/83948788-cc9ab480-a7ed-11ea-89d5-25361f871a31.png)

Should I use spacing, commas, or nothing for these?

**Comment by chochem** [comment]
The others all have EU in the name so I would add that.

I would go without spacing or commas as its the title. But, whatever you think looks best.

btw: that tab description is extremely well hidden with new tray layout (not a fan personally because lock tray doesnt even work with a restart). It took me a moment to find the description even after you mentioned that it still exists.

**Comment by Prometheus0000** [comment]
Changed to no spaces.

On another note, for the circuit imprints, are they the same whether you use the cheap or expensive (SoC or not) recipe to make the circuit? Because requiring you to make it again just to complete the quest would be bad (if you made one already and it works for either recipe)

**Comment by chochem** [comment]
yeah you can use a non-SoC circuit for the imprint and still do the SoC recipe

**Comment by Prometheus0000** [comment]
Guess I'll have to make it a checkbox task. Actually can't you OC the CAL, making the EV and IV ones LuV and ZPM?

**Comment by chochem** [comment]
you can OC the CAL, actually it has perfect OC I think.

> making the EV and IV ones LuV and ZPM?

I dont know what that is supposed to mean.

**Comment by Prometheus0000** [comment]
![image](https://user-images.githubusercontent.com/8517039/83949834-fa82f780-a7f3-11ea-92cf-7bfa3fbc78da.png)
![image](https://user-images.githubusercontent.com/8517039/83949838-02429c00-a7f4-11ea-8957-c296cfd9255a.png)

**Comment by chochem** [comment]
sorry, I have no idea what you are trying to say. These are the circuit assembler recipes, not the CAL. the circuit assemblers overclock normally but I dont see how that affects anything.

If you mean the CAL recipe, you might be talking about giving it 4A? in that case the answer is no. The CAL is limited to a single energy hatch.

**Comment by Prometheus0000** [comment]
> you can OC the CAL, actually it has perfect OC I think.

> If you mean the CAL recipe, you might be talking about giving it 4A? in that case the answer is no. The CAL is limited to a single energy hatch.

*Confused dog look*

I mean OCing the CAL so you can make first one in LuV with 2 LuV hatches.
The CAL recipes are the same power tier as the singelblock ones.

**Comment by Regian24** [comment]
> The CAL is limited to a single energy hatch.

**Comment by chochem** [comment]
one energy hatch only. OC = only with a higher tier energy hatch

**Comment by Prometheus0000** [comment]
OK. It still forms though, that should be changed.

*Is what I was going to say.* But I just tested it and it works.

**Comment by Prometheus0000** [comment]
Huh, I tested out the crystal one too but forgot to change the hatches. *But it still worked.* 
This is weird.

**Comment by chochem** [comment]
you are right. it actually works. I should not have believed bot!

**Comment by chochem** [comment]
updated the list. So the ASoC based Quantumprocessor recipe should be added to the LuV tab, and the Crystal SoC based Crystalprocessor should be in ZPM.

**Comment by Prometheus0000** [comment]
> If you do add "go to dim" quests then please make sure they are absolutely required, it is infuriating flying to a dimension to immediately leave just for a quest.

@GTNH-Colen Please tell me which ones are absolutely required then? Otherwise it's gonna be all of them.

**Comment by Prometheus0000** [comment]
> Tritanium. Lets make some Tritanium in the mk2 fusion reactor. useful for: T6 rocket, Gravisuit (has quest), possible tool material, drilling plant and void miners

Isn't tritanium luv? Also you need duranium to make it, should that have a separate quest or is it only used to get the tritanium?

Speaking of fusion materials, there's also: Califronium, Oganesson, Curium, Rhugnor, Flerovium, and Hypogen.

I don't think Rhugnor, Flerovium, Hypogen are used, but are the other 3?

Curium -> Californium ->Oganesson (Q or no? used in VM, but are the other 2 used elsewhere?)

**Comment by Regian24** [comment]
Bio questline
----------------------------------------------------
It's Not Going Bad, I Swear!
To make your next tier of circuits (Bio Circuits), you need to make Raw Bio Medium. There are three ways of making it.
The first one requires Stemcells, Unknown Crystal and T-Ceti Seaweed Extract at LuV voltage and 94 Sievert (Plutonium level) in a Bacterial Vat. It makes Raw Bio Medium at 4:1 ratio.
The second one is basically the first one, but it requires Infinity Catalyst instead of Tritanium at ZPM voltage and 140 Sievert (Enriched Naquadah level) in a Bacterial Vat. It makes Raw Bio Medium at 1:1 ratio.
The third one is the most efficient, but it requires Biocells (which you don't have yet), and 4 dust of Infinity Catalyst at UV voltage and 150 Sievert (Naquadria level) in a Bacterial Vat. It makes Raw Bio Medium at 1:2 ratio.


I'm Telling You, It's Not Mold!
Now let's make Biocells. To make them, you have to heat up your Raw Bio Medium to make Sterilized Bio Medium. 
Now you just have to use a Chemical Reactor to make Biocells.

Mutations are Good... Right?
Making the Bio Board is pretty much the same process as making a wetware board, just that... oh god that's expensive. 
To make the Ultra Bio Mutated Circuit Board, you just need to etch it with neutronium.

Self-Aware Circuits
Make a Bio Processing Unit. It is an important component in making the Bio Circuits.
Making it is an Assembly Line process at 160,000 EU/t, with IC2 Coolant, UU-Matter and Sterilized Bio Medum as fluid.

**Comment by chochem** [comment]
> 
> 
> > Tritanium. Lets make some Tritanium in the mk2 fusion reactor. useful for: T6 rocket, Gravisuit (has quest), possible tool material, drilling plant and void miners
> 
> Isn't tritanium luv? Also you need duranium to make it, should that have a separate quest or is it only used to get the tritanium?
> 
> Speaking of fusion materials, there's also: Califronium, Oganesson, Curium, Rhugnor, Flerovium, and Hypogen.
> 
> I don't think Rhugnor, Flerovium, Hypogen are used, but are the other 3?
> 
> Curium -> Californium ->Oganesson (Q or no? used in VM, but are the other 2 used elsewhere?)

T1 fusion only goes to 160MEU start-up cost, so tritanium is T2 and thus ZPM.

Duranium is pretty unimportant by comparison. I only remember using it for tritanium and for the gravichestplate (which needs tritanium anyway). You can still give it a quest after T1 fusion if you want.

regarding the other materials: Oganesson is certainly important for the VM. (VM3 recipe and 64x multiplier)  I mentioned it there, but maybe better a quest after T3 fusion?

I dont know of any uses for Curium or Californium outside of Oganesson tbh. Hypogen is T4 fusion, so if at all, it should be Endgame goals tab (but no uses yet afaik). Rhugnor is a future gt++ high tier plastic. no need to add a quest until it actually gets a use. And I know nothing at all about Flerovium.

**Comment by Prometheus0000** [comment]
![image](https://user-images.githubusercontent.com/8517039/83966326-91e55a80-a887-11ea-9fa4-4921b73e7943.png)
So the LuV is a lie? And are the other ones I listed also not what they say?

**Comment by chochem** [comment]
yeah. You always need to look at Voltage + startup cost both. Its quite misleading.

**Comment by chochem** [comment]
Actually good point: The quest for Mk1 and Mk2 fusion are quite clear on this. However, the one for Mk3 needs a change. The Mk3 is limited to 640MEU.

So hypogen, flerovium, the curium route for californium (it has a second recipe), are not possible in a Mk3 even though the quest says that`it can do *all fusion recipes*.

**Comment by Regian24** [comment]
i might be missing something, but how the heck do you get neutronium?

**Comment by Prometheus0000** [comment]
![image](https://user-images.githubusercontent.com/8517039/83966633-45028380-a889-11ea-94a5-c36a9417314b.png)

It says 200,000,000 at the bottom, do I look at the top number?

Small ores or fusion for Nt.

**Comment by Regian24** [comment]
Okay thank you

**Comment by chochem** [comment]
200 miilion > 160 million so no T1.

First small ore, later proper vein, fusion is not really viable for neutronium.

**Comment by Prometheus0000** [comment]
It says specifically on the spreadsheet's second page. They're tricky to remember/know about though. That's why I also mentioned them on the rocket quests for each of these tiers.

**Comment by Prometheus0000** [comment]
Oh, it's 10M *per energy hatch*, not total.

**Comment by Prometheus0000** [comment]
Don't forget there's also the cosmic SpNt version

**Comment by chochem** [comment]
![image](https://user-images.githubusercontent.com/40274384/83966718-d0910a00-a8b3-11ea-9c5d-fe8b336e9226.png)

ok maybe this isn't quite accurate either. Each hatch has a capacity of 10 MEU, so startup 160MEU or less.

**Comment by Prometheus0000** [comment]
I bet it was written by someone else who also hadn't done it. Anyway, just tell me exactly what to change and I will.

Also, the UV one says it explodes from too many amps?

**Comment by chochem** [comment]
in that picture (T1 fusion) it needs to say
> ..., and has a storage capacity of 10M eu.

T2 looks fine.

T3: remove the *this reactor is capable of doing all fusion recipes*. Replace by
> This reactor can handle recipes up to 640M eu startup.

I dont know about this exploding line. As it says *as usual* I assume it just means overvoltage.

**Comment by Prometheus0000** [comment]
I'm going over all 3, given all the little grammar problems, I assume Dream made them.

**Comment by Regian24** [comment]
Neutronium/Infinity Questline
----------------------------
The Power Of Neutrons (UV tab, because you need neutronium before T7 rocket)

Let's get started on this Avaritia stuff. To get Neutronium, go to space to find the small ore (Use NEI to find where it is :P), and mix it with Helium Plasma to make Raw Neutronium Dust.
It can be smelted in a Blast Furnace at 9000K of heat and 491,520 EU/t.
Quest requirement: 1 Neutronium Ingot

Neutrons From Space
Now let's make Cosmic Neutronium, a material that is used in a lot of late game stuff. 
Centrifuge Black Plutonium to get a Pile of Neutrons. It can then be compressed to Cosmic Neutronium Nuggets in an Implosion Compressor.
Compress the nuggets again to make a Cosmic Neutronium Ingot.
Quest requirement: 1 Cosmic Neutronium Ingot

Compressing the Core of a Star
Infinity Catalyst ingots need to be compressed with insane amounts of force. The only machine capable of doing that is a Neutronium Compressor. Let's make one.
Quest requirement: 1 Neutronium Compressor

A Glimpse of the Universe
The next material is Infinity Catalyst. First of all, go to Vega-B to get the dust. Then use a Neutronium Compressor to make Infinity Catalyst with 64 of the dust.
You can also do the insane avaritia recipe to make the Infinity Catalyst, if you want a challenge :P 
Quest requirement: 1 Infinity Catalyst

One Ingot, Infinite Power
Here at Questbook, Inc. we are seriously worrying about your health if you got here legitimately. Playing this much might be bad for you. You can always go outside, you know?
By the way, to make the ingot, it's just a simple Dire Crafting Table recipe.
Quest requirement: 1 Infinity Ingot

Collecting Neutrons the alternate way
Now that you have Infinity, you can use the Avaritia neutron collector to get Pile of Neutrons. It does not use any power.
Warning: This machine is extremely slow, so you might use something...
Quest requirement: 1 Neutron Collector

Breaking the Space-Time Continuum (optional quest)
Your new neutronium collecting device is insanely slow . It takes 3-5 minutes per pile of neutrons, so you have to wait over 5 hours if you want to get one ingot! 
But don't worry, there is a solution. You can make a Ultimate Time Anomaly (UV-Tier World Accelerator) to speed up the process by a lot. The maximum speed bonus you can get is 256x speed!
Quest requirement: 1 Ultimate Time Anomaly, 64 Pile of Neutrons
You can use

**Comment by chochem** [comment]
> The Power Of Neutrons (UV tab, because you need neutronium before T7 rocket)
> 
> Let's get started on this Avaritia stuff. To get Neutronium, go to space to find the small ore (Use NEI to find where it is :P), and mix it with Helium Plasma to make Raw Neutronium Dust.
> It can be smelted in a Blast Furnace at 9000K of heat and 491,520 EU/t.
> Quest requirement: 1 Neutronium Ingot

I put this at the end of ZPM for a reason. you need neutronium for UV components. you also really shouldnt mention avaritia in that quest, otherwise people will confuse this gregtech neutronium with the avaritia neutronium which is oredicted to cosmic neutronium. There is also no reason to discuss  where to find the ore as players will already have done Prometheus' Fusion Rocketry quest which tells you where to find neutronium small ore.

Better go with my initial suggestion and make this a quest at the end of ZPM that reads:

Neutronium for UV
A key ingredient for UV components as well as the T7 rocket is Neutronium from Gregtech. This is not be confused with Cosmic Neutronium or the Neutronium from Avaritia which you will encounter later on.

Mix your Neutronium dust with Helium Plasma to make Raw Neutronium Dust. It can then be smelted in a Blast Furnace at 9000K heat and 491,520 EU/t. This recipe takes a long time and a lot of power. Better make another EBF or Volcanus. Or a few. Or upgrade your Mega Blast Furnace.

Quest requirement: 64 Neutronium Ingot

**Comment by chochem** [comment]
> Compressing the Core of a Star
> Infinity Catalyst ingots need to be compressed with insane amounts of force. The only machine capable of doing that is a Neutronium Compressor. Let's make one.
> Quest requirement: 1 Neutronium Compressor

no ingots involved 
![image](https://user-images.githubusercontent.com/40274384/83967671-a3e0f080-a8bb-11ea-8917-324d094b9d65.png)

you might also want to mention singularities. Though they only have a very small number of late game uses like the infinity chest.

> Breaking the Space-Time Continuum (optional quest)

I dont like that one at all. this quest should not be in UHV. the machine is available at the start of UV and that is where the quest should be. Maybe I should have said it more clearly but this is not just for the neutron collector, its also really good for accelerating your IC2 crops, your fertilized dirt, your sprinklers, your industrial apiaries, ...

**Comment by Regian24** [comment]
> no ingots involved

I just mis typed lol

> this quest should not be in UHV
You can put it in UV if you want

**Comment by Regian24** [comment]
i'm sorry, all of this stuff has very similar names and it's confusing.


anyways, I fixed it

**Comment by Prometheus0000** [comment]
> New Data Bank quest: ask for 1 data bank, 1 assembly line slave connector, 1 data bank master connector, 18 computer heat vents, 6 high power casings. The number of computer casings and (advanced) data hatches is variable.
Also mention that it takes a maintenance hatch and at least an IV energy hatch. (it might be 2048 + (amount of master optics * amount of data hatches * 2048). scan to be sure)

Um, what does it do? Store more recipes for the assline? Can it be used with more than one? I saw a picture with cables going to asslines, I assume this is that? When you say scan do you mean build it and use a portable scanner on it? What are the cables to connect with? And _what_ does it connect to? Why are the computer casings and data hatches variable? Do you store data sticks in one of them?

Maybe you should just write the text?

**Comment by chochem** [comment]
each hatch position. can either have and advanced data hatch storing 16 sticks or a master connector going to a assembly line. mix as you like. there 12 or 16 or so hatch positions available, would need to check.

e.g. 8 assembly lines and 8x16= 128 data sticks for recipes (if it was 16 positions)

yes, scanning the controller will tell you exactly how much power you need. that formula is from bot, it seems to be correct I think.

energy hatches: normal gt cables

data bank master connector to assembly line slave connector: optical fiber cable that you need to color for it to work. you need to color the connector hatches too.

more data hatches=fewer computer casings.

data sticks go into the (advanced) data access hatch

> Maybe you should just write the text?
yeah I can do that

**Comment by chochem** [comment]
Quest: Data Bank
*optional quest*
requirements: the beginning of the end game

Description:

By now you probably feel overwhelmed with the number of data sticks for your assembly line. TecTech has a solution for you: The Data Bank. It allows you to store more than just 16 data sticks and can be connected to multiple Assembling Lines at the same time.

Place the controller one block above the ground and use the Multiblock Machine Blueprint to see how it is build. You will need a Maintenance Hatch and at least an IV Energy Hatch, though the power requirements can be higher if you use many Data Access Hatches and Data Bank Master Connectors. Scanning the controller will tell you how much is needed.

You will need more blocks then what is asked for in this quest, as the number of Data Access Hatches and Connectors is up to you. You can have, a maximum of 15 combined.

The Assembly Line Slave Connector replaces your old Data Access Hatch on the Assembling Line. Connect it to a Master Connector on the Data Bank with Optical Fiber Cable and make sure to color the hatches and the cable.

Retrieval Task:

1 Data Bank
6 High Power Casings
3 Computer Casings
1 Advanced Data Access Hatch
1 Data Bank Master Connector
1 Assembly line Slave Connector
10 Optical Fiber Cables

Quest Reward: 
3 Advanced Data Access Hatches

**Comment by Prometheus0000** [comment]
Does each master connector need a different color or is it just to prevent connecting to each other if they aren't spaced apart?

You forgot the 18 Computer Heat Vents. And verbalizing the blueprint's help note (I missed it for a while).

It doesn't seem to increase the amount of power usage stated in the portable scanner tooltip, even if I go from 1:1 to 1:4, 4:1, or 4:4, it just says 2048eu/t at 15 A.

Does it really use 15A of EV? That's just under 1 LuV, so how could an IV hatch power it, even with 2 amps?

**Comment by chochem** [comment]
> Does each master connector need a different color or is it just to prevent connecting to each other > if they aren't spaced apart?

I dont know tbh,

> You forgot the 18 Computer Heat Vents. And verbalizing the blueprint's help note (I missed it for a > while).

Yes, thanks. I had the heat vents on my list but they got lost when writing it down sorry.

> It doesn't seem to increase the amount of power usage stated in the portable scanner tooltip, even if I go from 1:1 to 1:4, 4:1, or 4:4, it just says 2048eu/t at 15 A.

>Does it really use 15A of EV? That's just under 1 LuV, so how could an IV hatch power it, even with 2 amps?

2048eu/t at 15 A. = 30000ish eu/t seems to work.

I dont think it really cares about voltage, only eu/t. overclocking in tectech needs a parametrizer to the best of my knowledge, not tried that though.

Does it really use that much? yes! thats why the quest says *at least an IV energy hatch*. I guess I have to clarify that :/

**Comment by Prometheus0000** [comment]
My main point about the power is that the tooltip doesn't say it increases even if I add more Advanced Data Access Hatches or Data Bank Master Connectors.

IV=8192, IV hatch takes 2A, 8192x2=16384 < 30,000? Did you mean an LuV hatch?

**Comment by chochem** [comment]
Thats why I wrote in the quest that it increases. And no, I meant *at least IV*. Not sure why this is unclear but I will try to reformulate it.

**Comment by Prometheus0000** [comment]
So I've been going over things and noticed some things in the tectech quests.
1) The uncertainty resolver has no recipe, but is apparently required for the Quantum Computer, and it's quest. Same for the UR X. Is there a non-standard recipe to get it?
2) As noted in #4707, you're required to get way more blocks than you need. I think reducing it to 4 each would be fine as long as the individual quests tell you what you need to complete them. And collecting everything at once would hold off the data bank quest (or future added ones).
3) Added a quest about painting, since even though it's mentioned you need to do so, it never tells you how.
4) What other tectech (finished) multis are there?
5) I read something about chaining quantum computers to increase computation ability or something?

**Comment by chochem** [comment]
> 
> 
> So I've been going over things and noticed some things in the tectech quests.
> 
>     1. The uncertainty resolver has no recipe, but is apparently required for the Quantum Computer, and it's quest. Same for the UR X. Is there a non-standard recipe to get it?
> 

Oh boy. That sounds like a major progression breaking bug. The uncertainty resolver most certainly had a recipe when I crafted mine in 9.0preX (or was is it in 8.9/8.8?). And yes, it is required. I dont known about the URX or what is does. Please make a ticket.



>     2. As noted in #4707, you're required to get way more blocks than you need. I think reducing it to 4 each would be fine as long as the individual quests tell you what you need to complete them. And collecting everything at once would hold off the data bank quest (or future added ones).
> 

Yeah, I pointed out the same in my overview. Collecting 4 each sounds like the better solution, I concur.

>     3. Added a quest about painting, since even though it's mentioned you need to do so, it never tells you how.

sounds fair.

> 
>     4. What other tectech (finished) multis are there?

no idea outside of these 3 and the transformer.

>     5. I read something about chaining quantum computers to increase computation ability or something?

yes. the blueprint design only allows for 2 computer racks. You have two options to increase that:
 1) extend the middle part. (just copy paste it a few times)  I am doing this. Noted it in my overview. It would be great if the quest would point out this option as it is already talking about 4 computer racks.
2) chaining them. The tectech wiki mentions that. I think Kharax does it in his videos. I have not done this. I guess the only question is how you connect them to each other, and my guess would be master connector/slave connector.

**Comment by Prometheus0000** [comment]
Ok, given that the Getting Europium quest title looks like this: "§b§9§d§l§b§c§lGetting Europium"
in code, I don't even want to try figuring that out for coloring the titles.

There should be some kind of normal way of adding colors, anyone know what it is?

**Comment by Regian24** [comment]
> There should be some kind of normal way of adding colors, anyone know what it is?

The text editor. You can try pasting some text from the text editor to the title, it may still retain the color.

**Comment by Prometheus0000** [comment]
Bleeeh. I'll let dream figure it out then. I also didn't add quest rewards either, since that would increase time to create significantly.

**Comment by chochem** [comment]
so first of all, the quests are awesome!

some issues at the moments (mostly minor):

**General:**

- BQ sound incorrect, compare https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/5078, https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/5342, though I don't know why this happens here.

**LuV:**

- Mysterious...:
typos *Enceledus* should be *Enceladus*, *macerate it* should be *macerate them* maybe also add the line: *You can also turn the dust back to unknown crystal shards with an autoclave.* (those shards is what you get from the crystals, they are used directly for Stemcells)

- Imprints, They're Important:
just turning it on should be enough to imprint the controller.

- Advanced SMDs
you could mention that they can also be used for low tier circuits at a 1ASMD : 4SMD ratio and that doing so also reduces the crafting time by 50%.

- Magneto Resonatic Circuits:
regarding *you can even make UEV circuits*. They exist alright, but you cant craft them. the UIV circuit assembler does not have a recipe. The UHV ones should be craftable though, but only after you made piko circuits for the UEV circuit assembler. Well, I guess piko circuits actually have some use then. 

**ZPM:**

- Nano Power ICs:
 the name exists twice. the first one is for Europium doped wafers.
- Your first Wetware Circuit:
Gooseberry Jelly. Is this a meme by bart or an incorrect item?
![image](https://user-images.githubusercontent.com/40274384/84069936-3d46fa00-a9c3-11ea-86a4-6813c68a7a46.png)
- same for Wet.Ass./Wet.Sup. (and yes, that is the correct abbreviation)

**UV**
- Quantum Computer (I know you didnt really get to these yet): Uncertainty Resolver: Schrödinger looks quite uncertain: 
![image](https://user-images.githubusercontent.com/40274384/84071103-11c50f00-a9c5-11ea-9495-3271f4228cec.png)
As already pointed out, the tectech wiki actually has a description on how this one works: https://github.com/Technus/TecTech/wiki/Uncertainty
it also contains this beautiful line:
> Disclaimer:: Sometimes it might be impossible to balance the focal points. Try waiting or replacing the hatch if u dare. (Yes this is a feature.)

   Other then that, we already talked about computer racks. the blueprint design only fits 2 computer racks, but the quest descriptions wants you to do 4 racks to get 288 computation. It should be pointed out that that needs an extension of the multiblock by replicating the middle section. (alternative: Daisy-chaining of 2+ Quantum Computers)

- Research Station: this one is actually fairly straight forward. Only things I would add is the info to put the controller 3 blocks above the ground, otherwise you miss part of the blueprint. Oh, and I heard it might be bugged in that it does not actually consume power (needs testing).`

- Which Doctor? THE Doctor: you should at least say the name. hard to look it up otherwise. So maybe: *If you haven't made a World Accelerator before, go...*`. Also, 4.2MEU/t = 8A, I assume? I dont think they use more than 6A. Machines can always accept a bit more than they use, to fill their internal buffer. So the tooltip about 8A is a bit misleading.

- compressing the core of a star and one ingot, infinite power:
should probably both be in UHV, but it depends how exactly the fix for the T8 rocket looks like, so wait on that. But in all likelihood it will need UHV energy hatches.

- Unlimited Power:
its not about *trying americium*, its more that the new T3 americium recipe giving plasma instead of molten fluid, so you kind of *have to* use that power to turn it into the molten americium. I dont even think there is an alternative way of cooling the plasma down.

well thats it for now, thanks for the nice work!

**Comment by Regian24** [comment]
> UEV circuit assembler.
That's a thing? I cant find it in NEI

**Comment by chochem** [comment]
![image](https://user-images.githubusercontent.com/40274384/84076321-71271d00-a9cd-11ea-999f-9dcf7aac97af.png)

**Comment by Regian24** [comment]
Huh, that's cool. I wonder what Pico Circuits are used for? Are they like "UIV Circuit"?

I don't think it's a good idea to make the UIV circuit just 2 of the UEV Circuit. I think it needs some sort of new line

**Comment by chochem** [comment]
nanos are inbetween UEV and Picos. the only other use for picos and the useless quantum circuits.
And I wouldn't worry about the details too much. this tiers are basically not implemented.

**Comment by Prometheus0000** [comment]
I don't think balancing piko circuits is anyone's priority. BTW have you tried using more than one blueprint at a time? You can see the larger versions easily.

**Comment by chochem** [comment]
aaahhhh, I see.
![image](https://user-images.githubusercontent.com/40274384/84077303-0b3b9500-a9cf-11ea-89be-5df60fab3108.png)



 *quantity effects tier* I always wondered what that means. I thought it was in line with *philosophers didn't even..*

**Comment by Prometheus0000** [comment]
> BQ sound incorrect, compare #5078, #5342, though I don't know why this happens here.

I could try find+replacing like moron, but I think finding the origin and fixing it is better. I suggest making a new issue and referring to the old ones (and the very specific one(s) in this one).

> Mysterious...:
typos Enceledus should be Enceladus, macerate it should be macerate them maybe also add the line: You can also turn the dust back to unknown crystal shards with an autoclave. (those shards is what you get from the crystals, they are used directly for Stemcells)

Fixed

> Imprints, They're Important:
just turning it on should be enough to imprint the controller.

Changed

> Advanced SMDs
you could mention that they can also be used for low tier circuits at a 1ASMD : 4SMD ratio and that doing so also reduces the crafting time by 50%.

About the halving time for ASMDs, it doesn't seem to always work properly (incorrectly set values?). See Quantumprocessor. And there was another one I forget that didn't half in the CAL but did in the singleblock.

> Magneto Resonatic Circuits:
regarding you can even make UEV circuits. They exist alright, but you cant craft them. the UIV circuit assembler does not have a recipe. The UHV ones should be craftable though, but only after you made piko circuits for the UEV circuit assembler. Well, I guess piko circuits actually have some use then.

Changed

> Nano Power ICs:
the name exists twice. the first one is for Europium doped wafers.

They were originally a combined quest that I broke apart. Fixed.

> Your first Wetware Circuit:
Gooseberry Jelly. Is this a meme by bart or an incorrect item?
same for Wet.Ass./Wet.Sup. (and yes, that is the correct abbreviation)

![image](https://user-images.githubusercontent.com/8517039/84078382-d61b4c80-a9a6-11ea-8277-f058ad1ded10.png)

Is this one of those problems that happens because of the ID numbers being different for people? @bartimaeusnek 

> Quantum Computer (I know you didnt really get to these yet): Uncertainty Resolver: Schrödinger looks quite uncertain:

Fixed by copying the character from wikipedia, let's hope that works.

I've been working on the tectech stuff, it's complicated after all.

> Which Doctor? THE Doctor: you should at least say the name. hard to look it up otherwise. So maybe: If you haven't made a World Accelerator before, go...`. Also, 4.2MEU/t = 8A, I assume? I dont think they use more than 6A. Machines can always accept a bit more than they use, to fill their internal buffer. So the tooltip about 8A is a bit misleading.

![image](https://user-images.githubusercontent.com/8517039/84079038-df58e900-a9a7-11ea-961a-ac3d6bef5e69.png)
![image](https://user-images.githubusercontent.com/8517039/84079063-e7188d80-a9a7-11ea-9bfb-cbaec190308b.png)
Looks like it uses 2x in TE mode, I'll adjust the text for the other quest as well. And say where it is.

> compressing the core of a star and one ingot, infinite power:
should probably both be in UHV, but it depends how exactly the fix for the T8 rocket looks like, so wait on that. But in all likelihood it will need UHV energy hatches.

Well that's annoying

> Unlimited Power:
its not about trying americium, its more that the new T3 americium recipe giving plasma instead of molten fluid, so you kind of have to use that power to turn it into the molten americium. I dont even think there is an alternative way of cooling the plasma down.

I meant more trying using it for power instead of Tin. Is it actually better than Tin? Should I change it to 'you can try other stuff' instead?

**Comment by Prometheus0000** [comment]
Actually, for the IC, since you have to go to Vega B to get Infinity Catalyst anyway, it'd be T8 even without assuming UHV hatches, right?

**Comment by chochem** [comment]
Oh yes, I had not seen the T8 rocket quest in UHV yet. yes, it should be after that rocket. I have not look at all tabs yet.

> I meant more trying using it for power instead of Tin. Is it actually better than Tin? Should I change it to 'you can try other stuff' instead?

more power density or more eu/t is irrelevant if you can produce the inputs fast enough. There is an overview in the datasheet and it looks like only mercury plasma has better eu/t (7.3mEU/t pre efficiency) and tin is second (5.4MEU/t). I dont know if this is upto date though. However tin makes it very easy to produce the inputs (silver->tin recycling in the molecular transformer + offworld pump for He-3). Americium is not great. Only a bit more than 1MEU/t, though I guess Hydrogen is easy and PU241 can be massproduced with trollcrops I think(?).

> I could try find+replacing like moron, but I think finding the origin and fixing it is better. I suggest making a new issue and referring to the old ones (and the very specific one(s) in this one).

I dont understand. Is this not something you set when making a quest? I guess I dont really know how that works, is there a place to set defaults?
The old issue was about the bq1 -> bq3 upgrading algorithm afaik, so that shouldnt come in here.

>About the halving time for ASMDs, it doesn't seem to always work properly (incorrectly set values?). See Quantumprocessor. And there was another one I forget that didn't half in the CAL but did in the singleblock.

I guess there is content for an extra ticket too. The 1:4 and the 50% is the intended behavior.

**Comment by Prometheus0000** [comment]
Does someone feel like writing some text to explain how the Parametrizer works? Because I don't really get it. Like, it configures stuff, but I don't understand what it's actually _doing._ I know it's supposed to work with the QC at least.

https://github.com/Technus/TecTech/wiki/Parametrization (There's 3 types now instead of two)
https://github.com/Technus/TecTech/wiki/Multiblock-base (You can see the lights where the effects are displayed)

@ScriptedPiky You wrote before you understood this thing

**Comment by chochem** [comment]
https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/6224

**Comment by Prometheus0000** [comment]
The sounds aren't set anywhere by me. Though if I go into Edit -> Advanced -> Object -> Object You can see:
![image](https://user-images.githubusercontent.com/8517039/84083397-88571200-a9af-11ea-91d9-2d3abc22b28b.png)
So it's probably some default setting somewhere is wrong.

**Comment by chochem** [comment]
> 
> 
> Does someone feel like writing some text to explain how the Parametrizer works? Because I don't really get it. Like, it configures stuff, but I don't understand what it's actually _doing._ I know it's supposed to work with the QC at least.
> 
> https://github.com/Technus/TecTech/wiki/Parametrization (There's 3 types now instead of two)
> https://github.com/Technus/TecTech/wiki/Multiblock-base (You can see the lights where the effects are displayed)
> 
> @ScriptedPiky You wrote before you understood this thing

I know very little. Namely: the quantum computer (and other multis I assume) have parameters that you can see by hovering over the lights in the controller GUI. I assume a parametrizer can change these values. E.g., you could probably force a overclock by changing at one of these: 
![image](https://user-images.githubusercontent.com/40274384/84083605-e0a30980-a9d9-11ea-9a76-9ca0ac362ce3.png)
![image](https://user-images.githubusercontent.com/40274384/84083626-e993db00-a9d9-11ea-84f2-902a21a7a92e.png)

**Comment by Prometheus0000** [comment]
That's about the level I understand it too. I tried changing something, and now I can't get it to not be undervolted, even though I just changed it back. I think. So I figured why not ask Piky, who said in a comment they understood it?

**Comment by chochem** [comment]
- UHV Energy hatch
why are you saying this is the last one? I guess they are not properly ordered in NEI. search for *accepts up to 2 Amps*. Or do you just mean that the higher tier ones dont have GTNH-style recipes yet, so arent fully implemented?

**Comment by ScriptedPiky** [comment]
So about the Parametrization. I may have forgotten how it truly works, but there is also a quest that *maybe* explains parts of it. I should really go back and rework that quest for up to date information. But if I recall correctly, the Parametrization allows you to modify a specific ID's to whatever you want; say, you want to overclock your Computer so it produces far more Computation, however, that may come at some rather dire consequences, or maybe it just overclocks to the next voltage (so if you have 20 computer casings, you might now use 20+A of UV or UHV instead of ZPM/UV!)

**Comment by Prometheus0000** [comment]
> UHV Energy hatch
why are you saying this is the last one? I guess they are not properly ordered in NEI. search for accepts up to 2 Amps. Or do you just mean that the higher tier ones dont have GTNH-style recipes yet, so arent fully implemented?

I forgot there were higher ones.

> So about the Parametrization. I may have forgotten how it truly works, but there is also a quest that maybe explains parts of it. I should really go back and rework that quest for up to date information. But if I recall correctly, the Parametrization allows you to modify a specific ID's to whatever you want; say, you want to overclock your Computer so it produces far more Computation, however, that may come at some rather dire consequences, or maybe it just overclocks to the next voltage (so if you have 20 computer casings, you might now use 20+A of UV or UHV instead of ZPM/UV!)

I understand that this is the purpose, but I don't understand what buttons to push to get that kind of result. It seems the first one adjusts which parameter you're adjusting after playing with it, but mine is still stuck undervolting.

> In order to precisely find out how much heat, you'll need a Parametrizer, which will be explained later.

It wasn't explained later though.

I'm going to go back over the quest myself, I just need to finish figuring out the multi. And doing things like look at https://github.com/Technus/TecTech/blob/61450ec14eabd637e1ce7af6690f3084ee6b2840/src/main/java/com/github/technus/tectech/thing/metaTileEntity/hatch/GT_MetaTileEntity_Hatch_Rack.java#L272 to understand the values of each circuit.

You can look it over again once I finish.

**Comment by Prometheus0000** [comment]
I made some text that _hopefully_ explains it sufficiently.
![2020-06-09_05 58 10](https://user-images.githubusercontent.com/8517039/84134221-41e8ce00-aa16-11ea-9a93-e6ef3f7914c3.png)
The below page is mostly borrowed from the wiki
![2020-06-09_05 58 16](https://user-images.githubusercontent.com/8517039/84134237-457c5500-aa16-11ea-87bd-def16f3fa009.png)

**Comment by chochem** [comment]
@Prometheus0000 I dont know if you have made quests for VM2-3 or are planning to do so. just wanted to say I was wrong orginally in where I put the VM2. I missed that it takes a research station. so it doesnt unlock together with T4 miner and VM1.

**Comment by Prometheus0000** [comment]
Since there's only a quest for the first ore drill I figured it'd be best for it to be the same for the void Miner.

**Comment by chochem** [comment]
I have updated to the latest file. Here is two more things:

- The new rocket quests and the new rocket fuel quests should be main quests (except for botmium casings obviously)
- Maybe it would be a good idea to give the *Chemical Plant Manual* as a quest reward for the *rocket fuel* quest (the info quest at the start of the rocket fuels). Just to make sure people actually have a look. Alternatively: Add the *Chemical Plant Manual* to the gt++ knowledge quest in the gt++ tab.

**Comment by Prometheus0000** [comment]
I'm not sure which ones you _absolutely have to do_ so I didn't make most of them main. @botn365 was complaining about being forced to make diesel instead of coal tar for the rp-1 quest, so I made the C-B D quests and the others non-main after adjusting it and some other quests. That's why the quest that previously required a canister is now a checkbox task, and was a C-B D cell in between. Could either of you tell me for sure which ones you can't skip?

Adding a note to the RF quest and putting it in the book one would probably be best.

**Comment by chochem** [comment]
the most obvious ones to make main quests are the actual rocket quests.

for the fuels its a bit more tricky I suppose:
Clearly you cant skip the green fuel, so everything that is needed for that can be main. But its possible you can skip any one of the previous fuels, it is not completely obvious.

**Comment by chochem** [comment]
I have had a first proper look ingame at the endgame tab and that still needs a good bit of work. I know there are two other issues on that, but you said you wanted all quest feedback here.

- remove the UEV circuit from the circuit quest 'these exist?' as that one has its own quest in the UHV tab now.

- The quest for a filled ultimate battery is just silly and should be removed without debate. Sorry, but clearly Colen did not do the math. If you have 10 fusion reactors just for filling that thing giving you 150M eu/t with infinity turbines which would be a crazy setup no one has ever done, then it would still take about 100 actual real life years to fill.

- Adding a quest for just the ultimate battery would be a good stepping stone to the really ultimate one

- the ultimate capacitor (kekztech) is now just a simple assembler recipe away from the really ultimate battery, so maybe not quite worth its own quest, but at least it should be mentioned in the quest for the really ultimate battery (extra quest is also fine of course)

- Also add a quest for the chaotic capacitor bank (infinite RF power). It would be a requirement for the Ridiculously Large Capacitor ('You'll Probably Need Quite a Few of These').

- The Stargate Radiation Containment Plate ('Building Block of the Stargate') should be another requirement for the Ridiculously Large Capacitor

- Eternal Singularity should be a requirement for the infinity chest.

- The Ridiculously Large Capacitor should be a requirement for the Stargate Command

- Stargate Command should be a requirement for the Teleporter ('Anywhere, Anytime')

**Comment by Prometheus0000** [comment]
I think we need another issue for 'changes/additions to endgame tab', with a link to the original issue, as it doesn't fit in with this issue. But yeah, I agree with your points, I'd just want to check ingame. If you make a new issue we can suggest adding new stuff like radox too without scaring people away from the size of the issue page.

**Comment by Prometheus0000** [comment]
> but you said you wanted all quest feedback here.

For the stuff covered by the LuV-UHV quest additions. Also, this can be considered done now honestly.
